packages/api/src/platforms/vtex/resolvers/searchResult.ts

```diff
@@ -41,6 +43,33 @@ export const StoreSearchResult: Record<string, Resolver<Root>> = {
 
     const facets = await is.facets(searchArgs)
 
-    return facets.attributes ?? []
+    const isCollectionPage = !searchArgs.query
+    const filteredFacets = facets?.attributes?.reduce((acc, currentFacet) => {
```

Comment:
`filteredFacets` doesn't mean much about the intention of this function. What would be the outcome of this filter?

Reply:
That's because we use this `reduce` function to filter just the `text` (`BOOLEAN`) filters (so far), and if the user was using the Collection page (PLP), we also remove the `departamento` facet, because the `PLP` represents the `departamento` already and this facet is not necessary on PLP. Finally, we also sort the facet values alphabetically.

*using `reduce` instead of `filter` so that we can use fewer loops.

As the main logic is a big filter, I thought about using `filteredFacets`, do you suggest another name that fits better?